doc/user/content/guides/postgres-cloud.md

```diff
@@ -123,6 +121,44 @@ As a user with the `cloudsqlsuperuser` role, make these changes to the upstream
 
 For more information, see the [Cloud SQL](https://cloud.google.com/sql/docs/postgres/replication/configure-logical-replication#configuring-your-postgresql-instance) documentation.
 
+## Azure Database for PostgreSQL
+
+Before you start, note that a database restart will be required after making changes to the database.
+
+1. The Materialize replica will need access to connect to the upstream database. In your Azure portal, go to the Azure Database for PostgreSQL instance and under the "Connections security" section add your [Materialize instance's IP address](/cloud/security/#static-ip-addresses) to the allowed IP addresses list and click on the "Save" button.
+
+1. Enable Logical Replication by going to the Azure Database for PostgreSQL instance, then under the "Replication" section click the "Enable Logical Replication" toggle button and click on "Save".
+
+1. Create a [publication](https://www.postgresql.org/docs/current/logical-replication-publication.html) with the tables you want to replicate:
+
+    ```sql
+    CREATE PUBLICATION mz_source FOR TABLE table1, table2;
+    ```
+
+     The `mz_source` publication will contain the set of change events generated from the specified tables, and will later be used to ingest the replication stream.
+
+## DigitalOcean Managed Postgres
```

Comment:
This section doesn't follow the structure of the other ones at all. Please use the same step-by-step format!

Reply:
Well, @ruf-io just asked me to get rid of those...

Reply:
Whatever we land on, it should just be consistent across the whole page.

Reply:
@ruf-io what is your take on this?